.github/workflows/release.yml

```diff
@@ -63,3 +63,178 @@ jobs:
         run: python -u ./dev/releases/make_github_release.py ${GITHUB_REF#refs/tags/} tmp/
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+  cygwin:
+    name: "Create Windows ${{ matrix.arch }} installer"
+    needs: unix
+    runs-on: windows-latest
+    # TODO Do we want to skip these jobs for pull requests?  If so, add the following line, and delete the PR-specific code from the steps below.
+    # if: ${{ github.event_name != 'pull_request' }}
+
+    strategy:
+      fail-fast: false
+      matrix:
+        arch:
+          - x86
+          - x86_64
+
+    steps:
+      - name: "Set git to use UNIX-style line endings"
+        run: |
+          git config --global core.autocrlf false
+          git config --global core.eol lf
```

Comment:
I have no idea, I've just been copy-and-pasting it around. I'll try commenting it out and we'll see what happens.

Reply:
Simply removing this steps causes the workflow to fail. In particular, the `release_gap.sh` script starts with:
```
:\a\gap\gap>SET CHERE_INVOKING=1 

D:\a\gap\gap>cd sage-windows 

D:\a\gap\gap\sage-windows>C:\cygwin64\bin\bash -l -c 'bash release_gap.sh' 
release_gap.sh: line 2: $'\r': command not found
<builtin>: target '.o' does not exist
cc      -o .o
cc: fatal error: no input files
compilation terminated.
make: *** [<builtin>: .o] Error 1
release_gap.sh: line 4: SAGE_GIT?=https://github.com/gap-system/gap: No such file or directory
release_gap.sh: line 5: SAGE_VERSION?=master: command not found
release_gap.sh: line 6: SAGE_MAKE_CONFIGURE_CMD?="cd $(SAGE_ROOT) && ./autogen.sh && ./configure": No such file or directory
release_gap.sh: line 7: SAGE_RUN_CONFIGURE_CMD?="cd $(SAGE_ROOT) && make && make bootstrap-pkg-full": command not found
release_gap.sh: line 8: $'\r': command not found
release_gap.sh: line 9: $'\r': command not found
release_gap.sh: line 10: $'\r': command not found
release_gap.sh: line 11: $'\r': command not found
release_gap.sh: line 12: $'\r': command not found
release_gap.sh: line 13: $'\r': command not found
release_gap.sh: line 14: SAGE_BUILD_DOC_CMD?="cd $(SAGE_ROOT) && make doc": command not found
release_gap.sh: line 15: $'\r': command not found
release_gap.sh: line 16: $'\r': command not found
release_gap.sh: line 17: $'\r': command not found
release_gap.sh: line 18: SAGEMATH_ISS?=gap.iss: command not found
release_gap.sh: line 19: $'\r': command not found
release_gap.sh: line 20: $'\r': command not found
release_gap.sh: line 21: $'\r': command not found
Error: Process completed with exit code 127.
```

According to https://github.com/actions/checkout/issues/135#issuecomment-613361104 we would need `.gitattributes` to contain `* text eol=lf` (it is currently `* text=auto`) but that fails: https://github.com/wilfwilson/gap/runs/2325655805?check_suite_focus=true 

In particular, the command `verify_git_clean()` of the `dev/releases/make_archives.py` script fails (presumably because this setting causes the line endings of some files to be changed when GAP is checked out?).

I'm not going to investigate this further.

Reply:
Thank you!